packages/@okta/vuepress-site/docs/guides/sign-into-spa/add-signin-button/react/login-redirect.md

```diff
@@ -1,48 +1,68 @@
-The `Auth` object is provided on your component's props via the `withAuth` component. You can use the `auth.isAuthenticated()` method to show or hide a button depending on whether the user is signed in.
+The `authState` object provides information on the state of the current user's authentiation.  
+
+The `authService` object provides methods to read more details about or to modify the current authentication.
+
+Function-based components can use the `useOktaAuth` [React hook](https://reactjs.org/docs/hooks-intro.html) to access either or both of the `authService` or the `authState` objects.
```

Comment:
Would omit "either or both of", and just say "to access the `authService` or the `authState` objects".